src/js/common/schemaform/GetPensionOrBurialFormHelp.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function GetFormHelp() {
+  return (
+    <div>
+      <p className="talk">For benefit related questions please call:</p>
+      <p className="phone-number">
+        <a href="tel:+1-800-827-1000">800-827-1000</a><br/>
+        Monday - Friday, 8:00 a.m. - 9:00 p.m. ET<br/>
+        <p className="talk">For Telecommunications Relay Service (TRS): dial <a href="tel:711">711</a></p>
```

Comment:
Thanks @cvalarida, really appreciate you highlighting this question, although this number should get folks through to TRS, perhaps additional instructions would bring help even closer within reach. Also interested in @J-Quag reflections on the best way to present this.